src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -74,12 +80,28 @@ const CooperationDetails = () => {
 
   const cooperationContent = activeTab && tabsData[activeTab]?.content
 
-  const pageContent =
-    isActivityCreated && activeTab === CooperationTabsEnum.Activities ? (
-      <CooperationActivities cooperationId={id} />
-    ) : (
+  const pageContent = () => {
+    if (
+      editMode &&
+      CooperationTabsEnum.Details === activeTab &&
       cooperationContent
-    )
+    ) {
+      return cooperationContent
+    } else if (editMode && activeTab === CooperationTabsEnum.Activities) {
+      return (
+        <CooperationActivitiesView
+          sections={response.sections}
+          setEditMode={setEditMode}
+        />
+      )
+    }
+
+    if (isActivityCreated) {
+      return <CooperationActivities cooperationId={id} />
+    } else {
+      return cooperationContent
+    }
+  }
```

Comment:
A better approach is to use guard clause here.

Here is useful video on how to use guard clause: https://www.youtube.com/shorts/6DPMVu6C70g

Reply:
Thanks, fixed